src/Followable.php

```diff
@@ -62,10 +123,17 @@ public function isFollowing($user)
 
         /* @var \Illuminate\Database\Eloquent\Model $this */
         if ($this->relationLoaded('followings')) {
-            return $this->followings->contains($user);
+            return $this->followings
+                ->filter(function ($following) {
+                    return $following->pivot->accepted_at !== null;
+                })
```

Comment:
```diff
- ->filter(function ($following) {
-            return $following->pivot->accepted_at !== null;
- })
+ ->whereNull('pivot.accepted_at')
```

Reply:
here it would need to be ->whereNotNull('pivot.accepted_at')

Reply:
Oh, I'm sorry! it is a typo.